shell/protoc.sh

```diff
@@ -64,6 +64,7 @@ docker exec "$CONTAINER_ID" sh -c "groupadd -f --gid $gid localuser && useradd -
 info_sub "go"
 docker exec --user localuser "$CONTAINER_ID" entrypoint.sh -f './*.proto' -l go \
   $(for import in $(get_list "go-protoc-imports"); do echo "-i /mod/$(get_import_basename "$import")"; done) \
+  $(if has_feature "validation"; then echo "--with-validator --validator-source-relative"; fi) \
```

Comment:
How will a user declare this feature?

Reply:
Per our conversation `has_feature "validation"` interacts with the `service.yaml` file driven by bootstrap (https://github.com/getoutreach/devbase/blob/main/shell/lib/bootstrap.sh). So it would be enabled in there with `validation: true`